server/routes/treeimages/treeimages-router.js

```diff
@@ -0,0 +1,47 @@
+import express from 'express';
+import AppError from '../../errors/AppError.js';
+import {
+  createImage,
+  deleteImage,
+  getAllImages,
+  getImageById,
+} from './treeimages-queries';
+import validatePostTreeLikes from './treeimages-validations';
+import { create } from 'domain';
+
+const treeimagesRouter = express.Router();
+
+//TODO change URL
+treeimagesRouter.get('/treeimage/:filenumber', async (req, res) => {
+  const { idImage,id,treeImages } = req.query;
+
+  if (!idImage && treeImages === 'All') {
+    let foundImages = await getAllImages();
+    if (!foundImages || foundImages.length === 0)
+      throw new AppError(400, 'Error getting Images');
+    res.status(200).json(foundImages);
+  }
+
+  if (idImage && id) {
+    const responseImage = await getImageById(idImage && id);
```

Comment:
@Spicytuna24 I'm a little confused, shouldn't you call `getImageById()` like this:
```ts
getImageById(idImage, id)
```
Assuming you can omit the filename from the arguments.